src/main/java/com/minecolonies/entity/ai/citizen/guard/AbstractEntityAIGuard.java

```diff
@@ -0,0 +1,327 @@
+package com.minecolonies.entity.ai.citizen.guard;
+
+import com.minecolonies.colony.buildings.AbstractBuilding;
+import com.minecolonies.colony.buildings.AbstractBuildingWorker;
+import com.minecolonies.colony.buildings.BuildingGuardTower;
+import com.minecolonies.colony.jobs.JobGuard;
+import com.minecolonies.colony.permissions.Permissions;
+import com.minecolonies.entity.ai.basic.AbstractEntityAISkill;
+import com.minecolonies.entity.ai.util.AIState;
+import com.minecolonies.entity.ai.util.AITarget;
+import com.minecolonies.tileentities.TileEntityColonyBuilding;
+import com.minecolonies.util.InventoryFunctions;
+import com.minecolonies.util.InventoryUtils;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.minecolonies.util.InventoryUtils'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)